src/actions/map.js

```diff
@@ -51,6 +52,12 @@ export const tOpenMap =
         try {
             const map = await fetchMap(mapId, dataEngine, keyDefaultBaseMap)
 
+            // record visualization view
+            dataEngine.mutate(dataStatisticsMutation, {
+                variables: { id: mapId },
+                onError: (error) => console.log('Error: ', error),
```

Comment:
Use same console.error as for useDataMutation(dataStatisticsMutation, ...) in FileMenu,js?